File: RemoteSigner/Models/ErrorCodes.cs
using System;
namespace RemoteSigner.Models {
    public static class ErrorCodes {
        public readonly static String InternalServerError = "INTERNAL_SERVER_ERROR";
        public readonly static String NotFound = "NOT_FOUND";
        public readonly static String EmailAlreadyInUse = "EMAIL_ALREADY_IN_USE";
        public readonly static String NoDataAvailable = "NO_DATA_AVAILABLE";
        public readonly static String InvalidLoginInformation = "INVALID_LOGIN_INFORMATION";
        public readonly static String NotLogged = "NOT_LOGGED";
        public readonly static String AlreadyExists = "ALREADY_EXISTS";
        public readonly static String PermissionDenied = "PERMISSION_DENIED";
        public readonly static String InvalidTokenType = "INVALID_TOKEN_TYPE";
        public readonly static String InvalidFieldData = "INVALID_FIELD_DATA";
        public readonly static String AlreadyClient = "ALREADY_CLIENT";
        public readonly static String AlreadyPaid = "ALREADY_PAID";
        public readonly static String PaymentError = "PAYMENT_ERROR";
        public readonly static String InsufficientFunds = "INSUFFICIENT_FUNDS";
        public readonly static String BankingSystemOffline = "BANKING_SYSTEM_OFFLINE";
        public readonly static String OutdatedAPI = "OUTDATED_API";
        public readonly static String BankNotSupported = "BANK_NOT_SUPPORTED";
        public readonly static String VaultSystemOffline = "VAULT_SYSTEM_OFFLINE";
        public readonly static String ServerIsBusy = "SERVER_IS_BUSY";
        public readonly static String Revoked = "REVOKED";
        public readonly static String AlreadySigned = "ALREADY_SIGNED";
        public readonly static String Rejected = "REJECTED";
        public readonly static String OperationNotSupported = "OPERATION_NOT_SUPPORTED";
        public readonly static String GraphQLError = "GRAPHQL_ERROR";
        public readonly static String OperationLimitExceeded = "OPERATION_LIMIT_EXCEEDED";
        public readonly static String InvalidTransactionDate = "INVALID_TRANSACTION_DATE";
        public readonly static String InvalidSignature = "INVALID_SIGNATURE";

    }
}

File: QuantoError/error_codes.go
package QuantoError

// region Public Use
const InternalServerError = "INTERNAL_SERVER_ERROR"
const NotFound = "NOT_FOUND"
const EmailAlreadyInUse = "EMAIL_ALREADY_IN_USE"
const NoDataAvailable = "NO_DATA_AVAILABLE"
const InvalidLoginInformation = "INVALID_LOGIN_INFORMATION"
const NotLogged = "NOT_LOGGED"
const AlreadyExists = "ALREADY_EXISTS"
const PermissionDenied = "PERMISSION_DENIED"
const InvalidTokenType = "INVALID_TOKEN_TYPE"
const InvalidFieldData = "INVALID_FIELD_DATA"
const AlreadyClient = "ALREADY_CLIENT"
const AlreadyPaid = "ALREADY_PAID"
const PaymentError = "PAYMENT_ERROR"
const InsufficientFunds = "INSUFFICIENT_FUNDS"
const BankingSystemOffline = "BANKING_SYSTEM_OFFLINE"
const OutdatedAPI = "OUTDATED_API"
const BankNotSupported = "BANK_NOT_SUPPORTED"
const VaultSystemOffline = "VAULT_SYSTEM_OFFLINE"
const ServerIsBusy = "SERVER_IS_BUSY"
const Revoked = "REVOKED"
const AlreadySigned = "ALREADY_SIGNED"
const Rejected = "REJECTED"
const OperationNotSupported = "OPERATION_NOT_SUPPORTED"
const GraphQLError = "GRAPHQL_ERROR"
const OperationLimitExceeded = "OPERATION_LIMIT_EXCEEDED"
const InvalidTransactionDate = "INVALID_TRANSACTION_DATE"
const BoletoCreationNotEnabled = "BOLETO_CREATION_NOT_ENABLED"
const BoletoOurNumberExausted = "BOLETO_OUR_NUMBER_EXAUSTED"
const NotImplemented = "NOT_IMPLEMENTED"

// endregion

// region Internal Use - Don"t worry about these if you"re a partner.
const EverythingIsTerrible = "EVERYTHING_IS_TERRIBLE"
const QuantoInternalError = "QUANTO_INTERNAL_ERROR"
const RoutingSystemOffline = "ROUTING_SYSTEM_OFFLINE"
const QITSystemOffline = "QIT_SYSTEM_OFFLINE"
const TargetConnectionError = "CONNECTION_ERROR"
const VaulterIsDead = "VAULTER_IS_DEAD"
const SynchronizationError = "SYNCHRONIZATION_ERROR"
const RoutingError = "ROUTING_ERROR"

// endregion
